service/src/main/java/edu/softserve/zoo/service/impl/WarehouseServiceImpl.java

```diff
@@ -20,6 +24,21 @@
     private WarehouseRepository warehouseRepository;
 
     @Override
+    @Transactional
+    public Warehouse update(Warehouse entity) {
+
+        boolean warehouseOverflow = entity.getAmount() > entity.getMaxCapacity();
```

Comment:
@skyfenko I thought about that.
But isn't that responsibility of Controller?

oh, of course, you're right.
Because Controller is not the only way to user Services. E.h., other services can use it too.

I'll fix that.